add-on/_locales/en/messages.json

```diff
@@ -279,6 +279,14 @@
     "message": "Redirect requests for IPFS resources to the Custom gateway",
     "description": "An option description on the Preferences screen (option_useCustomGateway_description)"
   },
+  "option_useSubdomains_title": {
+    "message": "Use Subdomains",
+    "description": "An option title on the Preferences screen (option_useSubdomains_title)"
+  },
+  "option_useSubdomains_description": {
+    "message": "Isolate content roots from each other by loading them from subdomains at *.localhost and creating a unique Origin for each CID, IPNS or DNSLink record. Requires a local go-ipfs 0.5.0 or later.",
+    "description": "An option description on the Preferences screen (option_useSubdomains_description)"
+  },
```

Comment:
On _Preferences_ screen the above labels look like this:

![2020-04-03--02-42-59](https://user-images.githubusercontent.com/157609/78312682-da655c80-7554-11ea-8ba6-d59600a730a3.png)

@jessicaschilling @autonome I suspect this could be phrased in a more human way, but this is the best I could do for next Beta release.  Suggestions welcome before we ship to Stable.

This is a description of a toggle that lets people to switch to old path-based gateway on `127.0.0.1` if they choose to do so for some reason. (disabling it stops HTTP proxy and swaps Custom Gateway URL to use raw IP)